### PR TITLE
[GSoC] Add guideline for students

### PR DESCRIPTION
**Pull request by JavierCVilla**

### CODE REVIEW

File: gsoc/students-guideline.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+# Guideline for Students
+
+### First steps
+
+Get started checking out the [Google Summer of Code Guides for students](https://google.github.io/gsocguides/student/) and the [timeline](https://developers.google.com/open-source/gsoc/timeline) of the program.
+
+### How to apply to CERN-HSF
+
+1. Take a look at the [list of projects](/activities/gsoc.html#projects-in-2019) participating for this year.
+  * There you will find a list of proposals for each project, their description and the contact information of their mentors.
```
Comment (graeme-a-stewart) [inline gsoc/students-guideline.md]: "description" -> "descriptions" (refers to propsals)

File: gsoc/students-guideline.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  * There is also a list with [all the project proposals](http://hepsoftwarefoundation.org/gsoc/2019/summary.html) sorted in alphabetical order.
+
+2. Once you have chosen one or more proposals that you like, please contact the corresponding mentors and they will guide you through the next steps.
+
```
Comment (graeme-a-stewart) [inline gsoc/students-guideline.md]: Formatting looks a bit odd, with more space between bullets 2 and 3 than between 1 and 2. Maybe try removing the blank lines?

File: gsoc/students-guideline.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+3. Mentors will typically ask you to complete a technical test to evaluate if you have the skills required for the project.
+  * Please note that the vast majority of our GSoC projects do not require any physics knowledge; in case of doubt, ask the mentors.
+  * If you do well in the tests, mentors will usually ask you to write a description for your project and help you with the submission.
```
Comment (graeme-a-stewart) [inline gsoc/students-guideline.md]: Is "project" the right word here? Shouldn't it be "proposal"?

File: gsoc/students-guideline.md (hunk 2 above)
Comment (JavierCVilla) [reply gsoc/students-guideline.md]: I agree, rather than removing blank lines, I have tried to add the same extra space in between. What do you think?

Comment (graeme-a-stewart) [reply gsoc/students-guideline.md]: Yes, looks better now

**Pull request merged**